### DISCUSSION TITLE
Is this the Next.js Spectrum replacement?

### DISCUSSION THREAD

**Discussion by martinbavio** [vercel/next.js#10436] [Help]
I'm curious, is the plan to replace the Spectrum community with this new Github tool?

I like that it would keep everything centralized, but at the same time feel like the feature is still kind of raw, which is expected since it's an experiment.

Also, is the plan for Discussions to be like a forum board with categories and everything?

**Comment by timneutkens** [comment, marked as answer]
Discussions will replace Spectrum for the Next.js community.
We're working with GitHub to provide feedback to shape this feature and all of your help (everyone that got invited so far) is needed to do so, so feel free to give feedback on the other repo as to what you'd like to see etc. They'd super appreciate it.

**Comment by martinbavio** [comment]
Thanks, @timneutkens!

**Comment by natedunn** [comment]
Definitely like the direction @timneutkens. Does this means "Discussions" will only be on a per-repo level? Would definitely like to see a Zeit (organizational level) discussions board that could tag different repos depending on the topic, but at this point it might not be possible.

**Comment by simpsoka** [comment]
Thanks @natedunn! Glad you like it :D Right now it works on a per-repo level - and we're actively developing the feature. Topics are definitely something we're discussing a lot. This is super helpful - so keep the feedback coming!

**Comment by natedunn** [comment]
@simpsoka Thanks for the quick response. Maybe you have a solution in mind already. :)  

The only thing I will add that is some Orgs might have very little overlap in terms of their repos/topics of discussions. But others might have much more. I might have a discussion in mind that involves both Next.js and Now but I am unsure of which repo's Discussions to bring it to. Or what if Zeit staff want to make announcement thread about something "off-topic/off-repo" such as new Zeit merch ideas? Where does that go?

In short: topics don't always mirror repos (or vice versa) but some folks' threads might include several ideas or topics (which may or may not have their own repos), so that is why I brought up the "organizational level" Discussions as a potential way to solve that.

Hope that makes sense. Thanks for working on this and good luck moving forward!

**Comment by simpsoka** [comment]
@natedunn completely makes sense. thank you so much!

**Reply by natedunn** [reply]
I like the new replies!

**Reply by becca** [reply]
Awesome, glad to hear you're enjoying the nested comments functionality ✨

**Comment by alystair** [comment]
Needs dark mode to really take over :D

**Reply by natedunn** [reply]
Random but I wouldn't be mad

**Comment by agustif** [comment]
> Needs dark mode to really take over :D

You can use a browser extension in the meantime, I guess if they do add Dark Mode it would be to github site-wide.

I've good experiencies with Dark Reader both in Chrome and Firefox, it is sadly a pay-only app for Safari

**Comment by Vadorequest** [comment]
Okay, so we've got emojis and upvotes, just noticed those, so used to upvote using "👍" I'd missed it.

@simpsoka @becca Something that would be quite useful are "starred" conversations (it could also be useful for issues). So that when we want to keep one (or many) particular issues always visible, we could star them (or "pin" them, but "pin" terminology is already used for officially pinned issues, so a different terms would avoid confusion)

Once I star a discussion, it's be highlighted in the "Discussions" homepage (https://github.com/zeit/next.js/discussions), like in the top. This could be a way to quickly go access often visited discussion, etc.

Also, a different color code for conversations I've read, or haven't read.
Also, a specific icon in those I've been mentionned, wrote a post, etc. Basically help the user quickly notice where he has interacted already could be useful.

I tried to reply to a conversation by email earlier, and even though I sent the email reply, my message didn't get displayed in the conversation, I had to write it again. I supposed this feature isn't implemented yet?

**Reply by simpsoka** [reply]
Definitely interesting - we've been talking a bit about conversation highlights, which is similar, but maybe a bit different. Highlights could be something that everyone sees - vs. starred could be a bookmark you're trying to get back to. I think a lot of this has to do with improving searchability though, too.

**Reply by becca** [reply]
Thanks for all this feedback, @Vadorequest! 

What is the reason you want to star a discussion? Is it a particular comment you want to keep coming back to? What's there? 

When you mention color coding of discussions and their `read` status are you referencing viewing these on `/zeit/next.js` and wanting more of a tight-knit integration with notifications there? Or are you talking about viewing notifications elsewhere? 

Regarding the email issue - I'll look into this! Thanks for raising this 🙇

**Reply by Vadorequest** [reply]
You're welcome :)

"starring/pinning" a discussion would be to find it back fast and have it accessible fast, even if it would display on the 3rd page for other people. 

![image](https://user-images.githubusercontent.com/3807458/76402951-70f29380-6384-11ea-8b06-a290db1a250d.png)

There are a lot of available unused space on desktop, and one could make use for some of it (left/right). It could also be displayed on top, similar to what gmail does with categories, like I have all my unread emails on top, and once this section is over then I see all my emails by the order I received them.

---

Regarding colors and status, I mean it for the same page:
![image](https://user-images.githubusercontent.com/3807458/76403124-be6f0080-6384-11ea-84aa-9041e1fe7ab9.png)

There are some interesting meta-data already visible (likes, type of discussion), we could add more like colors, and a glyph/icon depending whether we were mentionned, or took part in the dialog, etc.

Goal: Give overview of useful metadata through something visible, like glyphs/icons and colors.

**Reply by Vadorequest** [reply]
@becca

**Reply by becca** [reply]
awesome thanks for all this feedback and the quick response @Vadorequest !
